src/arc.rs

```diff
@@ -136,16 +136,16 @@ impl Shape for Arc {
         iter::once(PathEl::MoveTo(self.center + p0)).chain(self.append_iter(tolerance))
     }
 
-    /// Note: shape isn't closed so area is not well defined.
+    /// Note: shape isn't closed so area is not well-defined.
```

Comment:
```suggestion
    /// Note: shape isn't closed so area is not well defined.
```

https://english.stackexchange.com/questions/410283/hyphen-well-defined-vs-well-defined